src/geode/mesh/helpers/repair_polygon_orientations.cpp

```diff
@@ -329,17 +330,20 @@ namespace geode
         {
             builder.edges_builder().delete_isolated_edges();
         }
-        Logger::info( "Repair polygons orientations: ",
-            polygons_to_reorient.size(), " polygons reoriented" );
+        if( verbose )
+        {
+            Logger::info( "Repair polygons orientations: ",
```

Comment:
Should we add a verbose mode or set the message into a Logger::debug or trace so client code can remove this not very important message?

Reply:
Sending the message to `Logger::debug(..)` would be OK in the context of my use-case.

Reply:
@BotellaA I've modified the PR to the message to `Logger::debug` instead of adding a verbose flag, as you've proposed.